homeassistant/components/homekit/__init__.py

```diff
@@ -229,6 +230,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
                 CONF_EXCLUDE_DOMAINS: [],
```

Comment:
Maybe wrap this in BASE_FILTER_SCHEMA() since old config isn't going to have the new keys?

Reply:
Oh ok. I thought maybe there was a reason that wasn't being done right now so I followed the pattern. You've made me much more conscious of the call stack :) But yea, I can do that instead.

Reply:
Since its only being called once per config entry, its no big deal.  Its the tight loops that the call stack depth matters.